flink-filesystems/flink-azure-fs-hadoop/pom.xml

```diff
@@ -185,13 +185,6 @@ under the License.
 								</relocation>
 							</relocations>
 							<filters>
-								<filter>
-									<artifact>org.apache.flink:flink-hadoop-fs</artifact>
-									<excludes>
-										<exclude>org/apache/flink/runtime/util/HadoopUtils</exclude>
-										<exclude>org/apache/flink/runtime/fs/hdfs/HadoopRecoverable*</exclude>
-									</excludes>
-								</filter>
```

Comment:
Just trying to understand, why do we need this change?

Reply:
Since currently we are extending these HadoopRecoverable, this not being in the jar while I submit the job it is not able to pick it up. This was the initial problem that users were reporting too.

Reply:
Can I resolve this?

Reply:
Sure. Resolved.